sjb/config/test_cases/test_branch_cluster_operator_e2e.yml

```diff
@@ -0,0 +1,95 @@
+---
+parent: 'common/test_cases/origin_built_release.yml'
+overrides:
+  sync: []
+extensions:
+
+  actions:
+    - type: "script"
+      title: "setup prerequisites"
+      script: |-
+        sudo pip install openshift
+        cd /tmp
+        wget https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+        sudo yum -y install epel-release-latest-7.noarch.rpm
+        sudo yum --disablerepo="*" --enablerepo=epel -y update ansible
+        sudo env GOPATH=/data go get -u github.com/cloudflare/cfssl/cmd/...
+        mkdir -p /tmp/.aws
+        mkdir -p /tmp/.ssh
+    - type: "host_script"
+      title: "transfer aws credentials to remote host"
+      script: |-
+        scp -F ./.config/origin-ci-tool/inventory/.ssh_config ~/.aws/clusteroperator-credentials openshiftdevel:/tmp/.aws/credentials
+        scp -F ./.config/origin-ci-tool/inventory/.ssh_config ~/.ssh/devenv.pem openshiftdevel:/tmp/.ssh/libra.pem
```

Comment:
Isn't the SSH pubkey added to the instance by the cloud provider?

Reply:
Also, maybe for these instances we should use an ad-hoc or less widely shared keypair. Thoughts?

Reply:
@dobbymoodge the public key is added to the instance, but not the private one.
We need the private key in the test vm so *it* can ssh into the target cluster machines.

If you want to create a new key just for this, that's great. I'd only ask that you let me know the name and to place the private key somewhere on the jenkins filesystem so the job can get to it.